lib/modules/ens/index.js

```diff
@@ -179,12 +187,7 @@ class ENS {
 
   addENSToEmbarkJS() {
     const self = this;
-    // TODO: make this a shouldAdd condition
-    if (this.namesConfig === {}) {
-      return;
-    }
-
-    if ((this.namesConfig.available_providers.indexOf('ens') < 0) && (this.namesConfig.provider !== 'ens' || this.namesConfig.enabled !== true)) {
+    if (self.providerNotENS) {
```

Comment:
I think you can remove this because we still want to register the ENS provider, which would allow a Dapp author to do:
`EmbarkJS.Names.setProvider('ens')`